derailed_benchmarks.gemspec

```diff
@@ -29,15 +29,26 @@ Gem::Specification.new do |gem|
   gem.add_dependency "rack",            ">= 1"
   gem.add_dependency "rake",            "> 10", "< 14"
   gem.add_dependency "thor",            ">= 0.19", "< 2"
-  gem.add_dependency "ruby-statistics", ">= 4.0"
+  if RUBY_VERSION >= '3.0'
+    gem.add_dependency "ruby-statistics", ">= 4.0"
+  else
+    gem.add_dependency "ruby-statistics", ">= 2.1"
+  end
   gem.add_dependency "mini_histogram",  ">= 0.3.0"
   gem.add_dependency "dead_end",        ">= 0"
   gem.add_dependency "rack-test",       ">= 0"
+  gem.add_dependency "base64",          ">= 0"
+  gem.add_dependency "mutex_m",         ">= 0"
+  gem.add_dependency "bigdecimal",      ">= 0"
+  gem.add_dependency "drb",             ">= 0"
+  gem.add_dependency "logger",          ">= 0"
+  gem.add_dependency "ostruct",         ">= 0"
+  gem.add_dependency "ruby2_keywords",  ">= 0"
 
   gem.add_development_dependency "appraisal"
   gem.add_development_dependency "webrick",  ">= 0"
   gem.add_development_dependency "capybara",  "~> 2"
   gem.add_development_dependency "m"
-  gem.add_development_dependency "rails",     "> 3", "<= 7"
+  gem.add_development_dependency "rails",     "> 3", "< 7.1"
```

Comment:
seems to be unintentional, "<= 7" will resolve to exactly `7.0.0` which has a breaking bug